treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -43,11 +43,23 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity, pa
     extra_properties = {}
     for k in signature_props.keys():
         if k not in ['option_collection_hash', 'machine_platform',
-                     'test', 'suite']:
+                     'test', 'suite', 'has_subtests', 'parent_signature']:
             extra_properties[k] = signature_props[k]
 
     repository = Repository.objects.get(name=project_name)
 
+    if parent_hash:
+        # the parent PerformanceSignature object should have already been created
+        try:
+            parent_signature = PerformanceSignature.objects.get(
+                signature_hash=parent_hash)
+        except PerformanceSignature.DoesNotExist:
```

Comment:
I'm not sure if `PerformanceSignature.DoesNotExist` is actually a thing, but for the time being using this as a placeholder for whatever the exception would be.